src/app/admin/coupon/register/page.tsx

```
@@ -172,25 +324,28 @@ export default function CouponRegisterPage() {
                     <div className="flex flex-col gap-2">
                         <label className="text-sm text-sub-gray">수량</label>
                         <div className="flex gap-4 items-center">
-                            <input 
-                                type="text"
-                                className="p-3 border rounded-md"
+                            <NumberInput
+                                className="w-32"
+                                placeholder="수량을 입력하세요"
                                 value={couponStock}
                                 onClick={handleClick}
                                 onChange={handleCouponStockChange}
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)